mail_notification_custom_subject/readme/CONFIGURE.md

```diff
@@ -9,5 +9,9 @@
   >   template should apply in the notification emails sent by Odoo.
   > - The field **Subject Template** accepts
   >   [Jinja](https://jinja.palletsprojects.com/en/2.11.x/) expressions.
+  > - The field **Subject to replace** accepts
+  >   [Jinja](https://jinja.palletsprojects.com/en/2.11.x/) expressions.
   > - The field **Replace** specifies if the template should replace
   >   existing content or append to it.
+  > - The field **Partial Replacement** specifies if the template should
+  >   parcial replace existing content.
```

Comment:
Jinja was dropped from Odoo in https://github.com/odoo/odoo/pull/108764. Although the syntax is similar, I think these docs are misleading.